fuzzers/aflchurn/fuzzer.py

```diff
@@ -0,0 +1,120 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Integration code for AFLChurn fuzzer."""
+
+import shutil
+import subprocess
+import os
+
+from fuzzers import utils
+
+
+# '-fsanitize-coverage=trace-pc-guard'
+def prepare_build_environment():
+    """Set environment variables used to build targets for AFL-based
+    fuzzers."""
+    cflags = ['-fsanitize=address', '-fsanitize-address-use-after-scope']
+    utils.append_flags('CFLAGS', cflags)
+    utils.append_flags('CXXFLAGS', cflags)
+    utils.append_flags('ASAN_OPTIONS', ['abort_on_error=1', 'symbolize=0'])
+
+    os.environ['CC'] = '/afl/afl-clang-fast'
+    os.environ['CXX'] = '/afl/afl-clang-fast++'
```

Comment:
Hi, @inferno-chromium , I notice that the compiler ENV in AFL is 
```
os.environ['CC'] = 'clang'
os.environ['CXX'] = 'clang++'
```
Are the `clang(++)`  already linked to the `/afl/afl-clang-fast(++)`?

Reply:
no it is not, call the general build function first [e.g. https://github.com/google/fuzzbench/blob/master/fuzzers/aflsmart/fuzzer.py#L25] and then just overwrite the needed cc, cxx.

Reply:
Thanks a lot!